wip-tmpolaczyk-unhandled-intercept-v2.md

```diff
@@ -0,0 +1,84 @@
+<pre>
+  WIP: WIP-XXXX
+  Layer: Consensus (hard fork)
+  Title: Remove message argument from UnhandledIntercept RADON error
+  Authors: Tomasz Polaczyk <tomasz@witnet.foundation>
+  Discussions-To: `#dev-general` channel on Witnet Community's Discord server
+  Status: Draft
+  Type: Standards Track
+  Created: 2020-09-03
+  License: BSD-2-Clause
+</pre>
+
+
+## Abstract
+
+The `UnhandledIntercept` RADON error is used to represent RADON script execution errors that do not have a corresponding error code. To help with the identification of unhandled errors, the `UnhandledIntercept` error includes an error message. This message is included in the tally transaction, and it must be agreed upon all the nodes in the network. Therefore, any changes to this message may result in hard forks. This proposal removes the message argument to avoid any future problems.
```

Comment:
```suggestion
The `UnhandledIntercept` RADON error is used to represent RADON script execution errors that do not have a corresponding error code. To help with the identification of unhandled errors, the `UnhandledIntercept` error includes an error message. This message is included in the tally transaction, and it must be agreed upon by all the nodes in the network. Therefore, any changes to this message may result in hard forks. This proposal removes the message argument to avoid any future problems.
```